From 575c209cff29f39636d1a6417117539e2a4b2a2a Mon Sep 17 00:00:00 2001
From: Cerbero Build System <cerbero@gstreamer.org>
Date: Thu, 26 May 2016 03:18:48 +0530
Subject: [PATCH] Fix build failure due to usage of gets with GCC 6

The warn-on-use macro was backfiring and causing a failure just by existing.
Nothing in libiconv even uses gets().
---
 srclib/stdio.in.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/srclib/stdio.in.h b/srclib/stdio.in.h
index 473c84c..2990bef 100644
--- a/srclib/stdio.in.h
+++ b/srclib/stdio.in.h
@@ -695,7 +695,7 @@ _GL_CXXALIASWARN (gets);
 /* It is very rare that the developer ever has full control of stdin,
    so any use of gets warrants an unconditional warning.  Assume it is
    always declared, since it is required by C89.  */
-_GL_WARN_ON_USE (gets, "gets is a security hole - use fgets instead");
+//_GL_WARN_ON_USE (gets, "gets is a security hole - use fgets instead");
 #endif
 
 
-- 
2.7.4

